README.markdown

```diff
@@ -266,6 +266,10 @@ Rsync comment.
 
 File containing motd info.
 
+##### `pid_file`
+
+PID file. Defaults to /var/run/rsyncd.pid.
```

Comment:
can you please document the `'UNSET'` behaviour?

Reply:
Good idea - done.